### ISSUE TITLE
stacks options: auto-size window, hiding button "Add a new stack"

### ISSUE THREAD

**Issue by kernelplv** [stacks-task-manager/stacks#24]
![subject](https://i.imgur.com/wuE0Y22.png)

I need options to hide the button for adding a new column from the common space, and I also need an option to allow Stacks to expand themselves, which should lead to hiding the scrollbar.
Sorry for my English

thank you :)

**Comment by synapse** [comment]
Hey @kernelplv your request is not clear. Can you rephrase your request?

**Comment by kernelplv** [comment]
sure

**Comment by doctopus** [comment]
I support the idea of hiding the "Add new stack" column. Since this is a infrequently used option, mostly only while setting up your stacks, it is better to keep this option in the preference pane, or the three dot menu of each stack, just above the "Delete Stack" button saying "Add Stack".

**Comment by synapse** [comment]
Hey @kernelplv check version v0.2.10

**Comment by kernelplv** [comment]
> Hey @kernelplv check version v0.2.10
❤
